zulipterminal/ui_tools/boxes.py

```diff
@@ -1731,3 +1731,13 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
             if hasattr(self.panel_view, "log"):
                 self.panel_view.body.set_focus(0)
         return super().keypress(size, key)
+
+    def mouse_event(
```

Comment:
I am not sure if we should support this in the first place given that we mention in the FAQ that mouse support is limited. I think this could go in another PR if we decide to have it later.

Reply:
I think the keypress refactor would avoid this issue and mouse would work as normal without causing havoc because currently it does.

Reply:
I'm closing this as I'm hoping this would automatically get resolved